target/linux/ar71xx/patches-4.14/911-MIPS:_Remove_superfluous_check_for___linux__.patch

```diff
@@ -0,0 +1,52 @@
+From 1287533d3d95d5ad8b02773733044500b1be06bc Mon Sep 17 00:00:00 2001
```

Comment:
This should go to /target/linux/generic/backports-4.14/
This way all MIPS taregts will benefit from this patch.